docs/DeveloperGuide.md

```diff
@@ -302,11 +302,222 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given index is out of range
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ContaX shows an error message indicating that the given index is invalid.
 
-      Use case resumes at step 2.
+    * Use case resumes at step 2.
+
+* 3b. The given index is of invalid format
+
+    * 3b1. ContaX shows an error message indicating that the given command is of invalid format.
+
+    * Use case resumes at step 2.
+
+**Use case: Edit a person**
+
+**MSS**
+1. User requests to list persons
+2. ContaX shows a list of persons
+3. User requests to modify a specific person by specifying the corresponding index and parameter flags followed by the new values to change to
+4. ContaX edits the person accordingly
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 3a. The given index is out of range
+
+    * 3a1. ContaX shows an error message indicating that the given index is invalid.
+
+    * Use case resumes at step 2.
+
+* 3b. The given index is of invalid format
+
+    * 3b1. ContaX shows an error message indicating that the given command is of invalid format.
+
+    * Use case resumes at step 2.
```

Comment:
Similar to my comment above, I think this can be generalized to invalid command syntax.

Reply:
Noted fixed.